scripts/danger/clean/dev.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
+set -eu
+
 read -p "This will remove all directories and SQL tables. Are you sure? (y/*) " -n 1 -r
 echo
 
 if [[ "$REPLY" =~ ^[Yy]$ ]]; then
-  NODE_ENV=test npm run ts-node -- --type-check "./scripts/danger/clean/cleaner"
+  NODE_ENV=$(test npm run ts-node -- --type-check "scripts/danger/clean/cleaner")
```

Comment:
We set the `test` value to env variable `NODE_ENV`, that will be used by our NodeJS script so I think you should remove this :)

Reply:
OK, so actually, if I understand correctly what you're doing, I think my changes are correct.

Reply:
No, because with your version NODE_ENV will contain the result of the `test npm run ts-node...` script. We just want NODE_ENV to contain the string 'test'.

Reply:
Ah, OK, get it. :)